types/index.d.ts

```diff
@@ -227,7 +227,7 @@ export type DiffOptions = {
 export type diffInfo = {
   __old?: any
   __new?: any
-}
+} & LocaleMessages
```

Comment:
I think we should use union type `|`, because because `returnDiff` return value  has a different type depending on the logic

Reply:
Thanks! OK!

Reply:
@kazupon 
I fixed.
Please check!